client-app/src/pages/account/views/account-users/account-users.ts

```diff
@@ -150,4 +159,40 @@ export default class AccountUsers extends Vue {
       });
   }
 
+  confirmSuspendUser(user: User) {
+    this.$bvModal.msgBoxConfirm(i18n.t('account.users.confirm-suspend-modal.message', [ user.userName ]) as string, {
+      size: 'md',
+      buttonSize: 'md',
+      title: i18n.t('account.users.confirm-suspend-modal.title') as string,
+      okTitle: i18n.t('account.users.confirm-suspend-modal.ok') as string,
+      cancelTitle: i18n.t('account.users.confirm-suspend-modal.cancel') as string,
+      footerClass: ['p-2', 'flex-row-reverse justify-content-start'],
+      hideHeaderClose: false,
+      centered: true
+    })
+      .then(value => {
+        if(value) {
+          this.suspendUser(user.id!);
+        }
+      });
+  }
+
+  confirmUnsuspendUser(user: User) {
+    this.$bvModal.msgBoxConfirm(i18n.t('account.users.confirm-unsuspend-modal.message', [ user.userName ]) as string, {
+      size: 'md',
+      buttonSize: 'md',
+      title: i18n.t('account.users.confirm-unsuspend-modal.title') as string,
+      okTitle: i18n.t('account.users.confirm-unsuspend-modal.ok') as string,
+      cancelTitle: i18n.t('account.users.confirm-unsuspend-modal.cancel') as string,
+      footerClass: ['p-2', 'flex-row-reverse justify-content-start'],
+      hideHeaderClose: false,
+      centered: true
+    })
+      .then(value => {
+        if(value) {
+          this.unsuspendUser(user.id!);
+        }
+      });
+  }
```

Comment:
I would prefer you to avoid duplicates. Could you think about how to reach it?
The difference:
![image](https://user-images.githubusercontent.com/12604378/79189332-abe85b00-7e4b-11ea-975e-8994cda7d9f0.png)